apis/config/v1beta1/configuration_types.go

```diff
@@ -29,10 +29,6 @@ import (
 type Configuration struct {
 	metav1.TypeMeta `json:",inline"`
 
-	// Namespace is the namespace in which kueue is deployed. It is used as part of DNSName of the webhook Service.
-	// Defaults to kueue-system.
-	Namespace *string `json:"namespace,omitempty"`
```

Comment:
we can't just remove fields from an API, as that breaks backwards compatibility.

Still worth getting the namespace from the file though.